src/FunctionApps/python/tests/FhirServerExport/test_main.py

```diff
@@ -15,9 +14,11 @@
 
 @mock.patch("FhirServerExport.fhir.download_from_export_response")
 @mock.patch("FhirServerExport.fhir.export_from_fhir_server")
-@mock.patch.object(AzureFhirserverCredentialManager, "get_access_token")
+@mock.patch("FhirServerExport.fhir.AzureFhirserverCredentialManager")
```

Comment:
Another import flag (sorry if this is obnoxious, just trying to get ahead of changes I know will need to happen)